debezium-core/src/main/java/io/debezium/relational/RelationalTableFilters.java

```diff
@@ -25,6 +25,7 @@
     private final TableFilter tableFilter;
     private final Predicate<String> databaseFilter;
     private final String excludeColumns;
+    private final Predicate<TableId> schemaSnapshotPredicate;
```

Comment:
```suggestion
    private final TableFilter schemaSnapshotFilter;
```

Can you also add a JavaDoc comment to the field?